README.md

```diff
@@ -55,18 +56,21 @@ This plugin uses `<authConfig>` profile(s) in order to connect with `ldap server
         </property>
       </authConfig>
     </authConfigs>
+  </security>
 ```  
 * **Url:** Specify your ldap server URL
```

Comment:
Instructions for LDAPs? Importing SSL certs? Provide an example with a non-standard port `ldap://ldap.example.com:1234/foo/bar` or whatever is the URL that's expected?